conans/client/generators/text.py

```diff
@@ -43,6 +43,8 @@ def __init__(self, cpp_info):
         self.version = cpp_info.version
         self.name = cpp_info.get_name(TXTGenerator.name)
         self.rootpath = "%s" % cpp_info.rootpath.replace("\\", "/")
+        self.generatornames = "\n".join("%s:%s" % (k, v) for k, v in cpp_info.names.items())
+        self.generatorfilenames = "\n".join("%s:%s" % (k, v) for k, v in cpp_info.filenames.items())
```

Comment:
Better to use `=` as separator (more ini-like format)? Or is there any reason to use `:`?